cypress/integration/InstancesTest.ts

```diff
@@ -79,6 +69,95 @@ describe("Instances Test", () => {
         .ouiaId(newInstanceName, "PF4/TableRow")
         .should("not.exist");
     });
+
+    describe("Instance List pagination test", () => {
+      it("Select other page", () => {
+        cy.get("[aria-label='Go to first page']").should("be.disabled");
+        cy.get("[aria-label='Go to previous page']").should("be.disabled");
+        cy.get("[aria-label='Go to last page']").should("not.be.disabled");
+        cy.get("[aria-label='Go to next page']").should("not.be.disabled");
+
+        // Go to next page
+        cy.get("[aria-label='Go to next page']").first().click();
+        cy.get("[aria-label='Go to first page']").should("not.be.disabled");
+        cy.get("[aria-label='Go to previous page']").should("not.be.disabled");
+
+        // as currently count of instances are covered in 2 pages
+        cy.get("[aria-label='Go to last page']").should("be.disabled");
+        cy.get("[aria-label='Go to next page']").should("be.disabled");
+
+        // Go to previous page
+        cy.get("[aria-label='Go to previous page']").first().click();
```

Comment:
From my point of view this 'previous page' click is not needed, as we do not check the effect of the click action.

- We should either remove it
- Or we should add assertions like we have after clicking 'next page'

Reply:
I firstly added assertions for current page (first page) as 
`cy.get("[aria-label='Go to first page']").should("be.disabled");
        cy.get("[aria-label='Go to previous page']").should("be.disabled");
        cy.get("[aria-label='Go to last page']").should("not.be.disabled");
        cy.get("[aria-label='Go to next page']").should("not.be.disabled");`
        
Then move to next page and asserted for next page as 
`// Go to next page
        cy.get("[aria-label='Go to next page']").first().click();
        cy.get("[aria-label='Go to first page']").should("not.be.disabled");
        cy.get("[aria-label='Go to previous page']").should("not.be.disabled");

        // as currently count of instances are covered in 2 pages
        cy.get("[aria-label='Go to last page']").should("be.disabled");
        cy.get("[aria-label='Go to next page']").should("be.disabled");`
        
and then move to previous page (which is the same as first page)

Reply:
`and then move to previous page (which is the same as first page) for which we already did the validation`

I agree that previous page is same as first page, where you did some assertions already. However the operation (last line) in your test:
`cy.get("[aria-label='Go to previous page']").first().click();`

How are we sure that after that clic really the fist page is shown? Are we sure there is never ending loading screen? or some other issue?

Reply:
I agree with your on this point. Will add the assertions for the same